app/src/main/java/com/hamdan/forzenbook/view/login/MainLogin.kt

```diff
@@ -111,12 +111,10 @@ private fun UserInputtingContent(
     )
     Spacer(modifier = Modifier.height(PaddingValues.medPad_2))
     val navController = LocalNavController.current
-    Text(text = "Create Account", modifier = Modifier.clickable {
-        navController?.navigate(NavigationDestinations.CREATE_ACCOUNT)
-    })
+    Text(
+        text = "Create Account",
```

Comment:
XML strings

Reply:
Right was gonna handle that in the UI Ticket but I think its fine to do it here just to have it dealt with then.